runtime/jvmti/jvmtiThread.c

```diff
@@ -1217,12 +1189,12 @@ ownedMonitorIterator(J9VMThread *currentThread, J9StackWalkState *walkState, j9o
 	j9object_t obj = (NULL != slot) ? *slot : NULL;
 
 	if (count >= (UDATA)walkState->userData4) {
-		/* PMR 69633,001,866 / CMVC 197115 
+		/* PMR 69633,001,866 / CMVC 197115.
 		 * There is a race condition that target thread actually acquires monitor(s)
 		 * between first and second pass (monitor enter is done outside of vm access).
-		 * userData4 is initialized to UDATA_MAX
-		 * 	1) at first pass, count will never be bigger than UDATA_MAX
-		 * 	2) at second pass, userData4 now holds the 'count' value obtained from first pass
+		 * userData4 is initialized to UDATA_MAX.
+		 * 	1) At first pass, count will never be bigger than UDATA_MAX.
+		 * 	2) At second pass, userData4 now holds the 'count' value obtained from first pass.
```

Comment:
Please expand the tabs within the comment.

Reply:
I didn't know how to expand the tabs. So, I just got rid of them.